bokeh/server/tests/test_server.py

```diff
@@ -167,7 +167,7 @@ def test__lifecycle_hooks():
     application = Application()
     handler = HookTestHandler()
     application.add(handler)
-    with ManagedServerLoop(application, check_unused_sessions_milliseconds=20) as server:
+    with ManagedServerLoop(application, check_unused_sessions_milliseconds=30) as server:
```

Comment:
@havocp This is the only change I wonder about. Was 20 ms supposed to be a tight bound? I had to raise to 30 to get passing on Windows. Otherwise session close and periodic callback were swapped